build.cmd

```diff
@@ -1,5 +1,12 @@
 @echo off
+.paket\paket.bootstrapper.exe
```

Comment:
oh that's what you meant. I use paket all the time on a lot projects, it's never been an issue for me.